include/zephyr/bluetooth/gatt.h

```diff
@@ -981,6 +981,9 @@ struct bt_gatt_notify_params {
 	bt_gatt_complete_func_t func;
 	/** Notification Value callback user data */
 	void *user_data;
+#ifdef CONFIG_BT_EATT
+	enum bt_att_chan_option chan_option;
```

Comment:
Wouldn’t “preference” be a better term than “option”? The struct member would then be called eg `chan_ pref`

Reply:
"preference" is a better name for the current use. "option" is better if we want to generalize this field in the future.

Reply:
If you want to make it completely generic, just call it `flags` or `opts`, however we could also cross that bridge when we get there (if it ever happens) and go with the more intuitive and specific naming for now.

Reply:
Changed to "preference".